### ISSUE TITLE
Does not pickup default node version

### ISSUE THREAD

**Issue by benmarten** [derekstavis/plugin-nvm#1]
I am using your wrapper - thanks for making it.

Only problem I have is that my fish shell keeps forgetting the default node version that I have set via:
`nvm alias default 0.12`

```
⋊> ~ node -v                                                                                                            16:59:06
fish: Unknown command 'node'
⋊> ~ bash                                                                                                               16:59:08
bash-3.2$ node -v
v0.12.7
bash-3.2$ exit
⋊> ~ nvm alias default 0.12                                                                                             16:59:22
default -> 0.12 (-> v0.12.7)
⋊> ~ node -v                                                                                                            16:59:28
v0.12.7
```

It is working in bash... any ideas?

**Comment by derekstavis** [comment]
Thanks for reporting the issue! I've fixed it in commit fcb2516. Upgrade the plugin using:

``` fish
omf update
```

I had forgotten to call nvm's initialisation code :stuck_out_tongue:

**Comment by benmarten** [comment]
Works like a charm now. Thanks for the fast update!
